vit/application.py

```diff
@@ -360,6 +369,21 @@ def command_bar_keypress(self, data):
                     # before hitting enter?
                     if self.execute_command(['task', metadata['uuid'], 'modify'] + args, wait=self.wait):
                         self.activate_message_bar('Task %s modified' % self.model.task_id(metadata['uuid']))
+                elif op == 'modify_multiple':
+                    # same underlying command is the modify command above, only the results is parsed
```

Comment:
`is parsed` or `are parsed`? I could see an argument either way.

Reply:
Actually nothing is parsed, `execute_command()` just returns a boolean, so the only difference is that upon successful execution `modify_multiple` doesn't try to retrieve the updated task's information through tasklib via its uuid. Updating comment accordingly.